packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -3294,6 +3295,17 @@ export interface VideoStreamRendererViewState {
 // @public
 export const VideoTile: (props: VideoTileProps) => JSX.Element;
 
+// @beta
+export interface VideoTileContextualMenuProps {
+    type: 'contextual';
+}
+
+// @beta
+export interface VideoTileDrawerMenuProps {
+    hostId?: string;
```

Comment:
context menu also has a hostId, would be good to have this for VideoTileContextualMenuProps also

Reply:
Good point but I will leave it off for now and wait for customer signals.